### ISSUE TITLE
The future of Mull-Fenix and mullvad-browser

### ISSUE THREAD

**Issue by matte22ladde** [Divested-Mobile/Mull-Fenix#183]
Both are Gecko based browsers that start with the word "Mull*" and there is an open issue for Android build request.
mullvad/mullvad-browser#24
Is there a room for a merge?

**Comment by SkewedZeppelin** [comment]
No, an Android version would be based on Base Browser like Tor Browser for Android.

**Comment by SkewedZeppelin** [comment]
That isn't the same thing.

Mull benefits from whatever work Tor Project has upstreamed into Firefox.
Tor Browser and its corresponding Base Browser still have numerous additional benefits over it.
